packages/ui/components/ui/card.tsx

```diff
@@ -1,26 +1,28 @@
 import * as React from 'react';
 import { cn } from '../../lib/utils';
+import { motion } from 'framer-motion';
 
 export interface CardProps extends React.HTMLAttributes<HTMLDivElement> {
   gradient?: boolean;
   light?: boolean;
+  layout?: boolean;
 }
 
 const Card = React.forwardRef<HTMLDivElement, CardProps>(
-  ({ className, gradient, light, children, ...props }, ref) => {
+  ({ className, gradient, light, children, layout, ...props }, ref) => {
     const baseClasses = 'rounded-lg shadow-sm p-[30px] overflow-hidden';
     return (
-      <div
+      <motion.div
+        layout={layout}
         ref={ref}
         className={cn(
           baseClasses,
           light ? 'bg-stone-300' : gradient ? 'bg-card-radial' : 'bg-charcoal',
           className,
         )}
-        {...props}
       >
-        {children}
-      </div>
+        <div {...props}>{children}</div>
```

Comment:
Had to pass `...props` to a child div rather than to `motion.div` because of differences in typings for e.g., `onAnimationStart`

Reply:
This has an effect on the grid layout (gap is no longer respected on send form). Think we can kill props entirely.